nixos/modules/services/x11/desktop-managers/none.nix

```diff
@@ -1,7 +1,46 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let
+  runXdgAutostart = config.services.xserver.desktopManager.runXdgAutostartIfNone;
+in
 {
-  services.xserver.desktopManager.session =
-    [ { name = "none";
-        start = "";
-      }
-    ];
+  options = {
+    services.xserver.desktopManager.runXdgAutostartIfNone = mkOption {
+      type = types.bool;
+      default = true;
```

Comment:
For all people that already uses something else (e.g.: `dex`), having this a default can end up the service being started twice right?

If so, I think it is better to either leave this option as false by default or to have an entry in the release notes. Actually an entry on the release notes is good even if this option is leave as default for false to help discovery.

Reply:
Also, maybe defaulting to false here is better anyway because I am not sure if people expect to having an auto start manager even when they don't use a a desktop manager.